Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
Do not use latest always fix a tag.

Reply:
The idea here is to stick to use the latest Red Hat UBI 7 i.e. it won't pull other RHEL specified version.

Reply:
We should never pull latest, that is how containers are built. We should always stick to a tag.